### DISCUSSION TITLE
Kubernetes Network Tuning for EMQx

### DISCUSSION THREAD

**Discussion by spokc** [emqx/emqx#4635] [Q&A]
Hi all,

EMQx has fantastic Linux Tuning guide https://docs.emqx.io/en/broker/v4.3/tutorial/tune.html#tcp-network-tuning. Do we have a similar setup achieving the same for kubernetes ? I'm currently trying to use `initContainers` to run sysctl commands but to no avail

**Comment by spokc** [comment]
Do we apply these settings on k8s Nodes directly as a cluster admin ? or any other way ?

**Reply by qzhuyan** [reply]
Have you checked pod security context? 
https://kubernetes.io/docs/tasks/configure-pod-container/security-context/

**Reply by spokc** [reply]
Yes, but we can't tweak all. It's very limited. Trying to set sysctls on the nodes and see if container namespaces honour it
